<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushBtnReadCurrentDataAndTime">
         <property name="text">
          <string>读取当前日期时间</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>字符串显示</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelTime">
         <property name="text">
          <string>时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushBtnSettingTime">
         <property name="text">
          <string>设置时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDate">
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushBtnSettingDate">
         <property name="text">
          <string>设置日期</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelDateTime">
         <property name="text">
          <string>日期时间</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushBtnSettingDateTime">
         <property name="text">
          <string>设置日期时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditTime"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditDate"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEditDateTime"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelSelectDate">
         <property name="text">
          <string>选择的日期</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushBtnClose">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>定时器</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushBtnTimerStart">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushBtnTimerStop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelOutTime">
           <property name="text">
            <string>流失时间：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>定时周期</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBoxTimerCycle">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="prefix">
            <string>0</string>
           </property>
           <property name="minimum">
            <number>500</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushBtnSettingCycle">
           <property name="text">
            <string>设置周期</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLCDNumber" name="lcdNumMinute"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumSecond"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumMillisecond"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
